.github/workflows/zkp_auth.yml

```
@@ -204,3 +204,30 @@ jobs:
+
+  deploy-infra:
+    name: Deploy Infrastructure to AWS
+    runs-on: ubuntu-latest
+    needs: [test-images]  # Deploy only after testing is complete
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

actionlint

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)